data-fundamentals/README.md

```diff
@@ -17,6 +18,22 @@ The goal of this section is to briefly describe how to apply the fundamentals to
 - [Observability](#observability)
 - [End to End and Azure Technology Samples](#end-to-end-and-azure-technology-samples)
 
+## Concurrency Control
+
+Your systems should (almost) always leverage some form of concurrency control, to ensure correctness amongst competing requests and to prevent data races. The 2 forms of concurrency control are Pessimistic and Optimistic.
+
+A pessimistic transaction involves a first request to "lock the data", and a second request to write the data. In between these requests, no other requests touching that data will succeed. See 2 [Phase Locking](https://en.wikipedia.org/wiki/Two-phase_locking) (also often known as 2 Phase Commit) for more info.
```

Comment:
clarity/readability: the full phrase "2 Phase Locking" should be used for the link

Reply:
done!